bindinfo/handle.go

```diff
@@ -448,6 +448,9 @@ func (h *BindHandle) Size() int {
 
 // GetBindRecord returns the BindRecord of the (normdOrigSQL,db) if BindRecord exist.
 func (h *BindHandle) GetBindRecord(hash, normdOrigSQL, db string) *BindRecord {
+	if record := h.bindInfo.Load().(cache).getBindRecordWithDB(hash, normdOrigSQL, db); record != nil {
```

Comment:
Why not combine it with `getBindRecord`?

Reply:
Done.